Backend/app/dashboard/CommonAPIs/spec/RiderPlatform/Management/API/MultiModal.yaml

```
@@ -0,0 +1,63 @@
+
+apis:
+  - POST: # PreprocessFRFSDataAPI
+      endpoint: multimodal/frfs/data/preprocess
+      auth: ApiAuthV2
+      params: # city inferred from api path variable
+        vehicleType: VehicleCategory
+        fileFormat: FileFormat
+        file: FilePath
+        inputDataType: RawDataType
+      response:
+          type: PreprocessFRFSDataResp
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)